packages/components/organisms/f-checkout/src/components/_tests/Error.test.js

```diff
@@ -1,26 +1,56 @@
 import { VueI18n } from '@justeat/f-globalisation';
 import { shallowMount, createLocalVue } from '@vue/test-utils';
+import Vuex from 'vuex';
 import Error from '../Error.vue';
-import { i18n } from './helpers/setup';
+import { i18n, $cookies, $logger } from './helpers/setup';
+import { CHEKOUT_ERROR_FORM_TYPE, VUEX_CHECKOUT_MODULE } from '../../constants';
 
 const localVue = createLocalVue();
 
 localVue.use(VueI18n);
+localVue.use(Vuex);
+
+const mockState = {
+    basket: {
+        id: 'menu-sentinel-6-freilich',
```

Comment:
I know this is a test, but could we please try to use a similar-to-prod example?

Reply:
👍

Reply:
Done.